apis/core/lib/domain/csv-source/delete.ts

```diff
@@ -15,12 +15,18 @@ export async function deleteSource({
   resource,
   store = resourceStore(),
 }: DeleteSourceCommand): Promise<void> {
+  await deleteSourceWithoutSave(resource, store)
+
+  // Save changes
+  await store.save()
+}
+
+export async function deleteSourceWithoutSave(resource: NamedNode<string>, store: ResourceStore): Promise<void> {
```

Comment:
I think it is preferable that the store.save is called only once when deleting a project.